widget/embedded/src/types/notification.ts

```diff
@@ -0,0 +1,21 @@
+import type {
+  Route,
+  RouteEvent,
+  Step,
+  StepEvent,
+} from '@rango-dev/queue-manager-rango-preset';
+
+export type Notification = Pick<
+  Step,
+  | 'fromBlockchain'
+  | 'fromSymbol'
+  | 'fromSymbolAddress'
+  | 'toBlockchain'
+  | 'toSymbol'
+  | 'toSymbolAddress'
+> &
```

Comment:
I think we can approve notification type :
type Route = {
  from: {
    blockchain: string;
    tokenSymbol: string;
    tokenAddress: string;
  };
  to: {
    blockchain: string;
    tokenSymbol: string;
    tokenAddress: string;
  };
}; 

Type Notification  = { route: Route , ...}